packages/shared/src/hooks/usePostModalNavigation.ts

```diff
@@ -18,15 +18,83 @@ export const usePostModalNavigation = (
   items: FeedItem[],
   fetchPage: () => Promise<unknown>,
 ): UsePostModalNavigation => {
+  const [currentPage, setCurrentPage] = useState<string>();
+  const isExtension = !!process.env.TARGET_BROWSER;
   const [openedPostIndex, setOpenedPostIndex] = useState<number>(null);
   const [isFetchingNextPage, setIsFetchingNextPage] = useState(false);
   const { trackEvent } = useContext(AnalyticsContext);
 
+  const changeHistory = (data: unknown, title: string, url: string) => {
+    if (!isExtension) {
```

Comment:
Is this method not supported in the extension? (Don't we do something similar with the feed types?)